samples/csharp_dotnetcore/45.state-management/StateMangementBot.csproj

```diff
@@ -7,7 +7,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.6.0-preview-191001-1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.6.0-preview-191001-1" />
```

Comment:
I'm still C#-dumb, but should the sample use the preview packages? It would necessitate users add the MyGet feed, wouldn't it? If not, it's fine.

I know we don't have `CosmosDbPartitionedStorage` in the current NuGet packages and the bot won't run without them...this is more of a pre-merge comment.

Reply:
@mdrichardson You are correct. However, I am merging all of my 4.6 samples, once approved, into a single branch that tracks the 4.6 MyGet preview packages. Once the public packages are available, I will update the one branch and PR to master.